app/policies/supply_policy.rb

```diff
@@ -0,0 +1,5 @@
+class SupplyPolicy < ApplicationPolicy
+  def manage_master_supply_list?
+    user.admin?
+  end
```

Comment:
this method could probably just be written as
```ruby
def manage_master_supply_list?
  user.admin?
end
```
because `user.admin?` will return `true` or `false`